org.eclipse.xtext.purexbase.eclipse.tests/META-INF/MANIFEST.MF

```diff
@@ -17,7 +17,8 @@ Require-Bundle: org.eclipse.xtext.purexbase,
  org.eclipse.xtext.xbase.lib,
  org.eclipse.xtext.testing,
  org.eclipse.xtext.xbase.testing,
- org.eclipse.xtext.ui.testing
+ org.eclipse.xtext.ui.testing,
+ org.eclipse.xtext.junit4
```

Comment:
maybe we need a change to the generator to skip this.

Reply:
yes, but that is what the generator currently produces. doesn't harm, but can be improved.

Reply:
yes this is why i propose to make this choosable in the generator.